tools/templates/iiot_deployment_mac.yml

```diff
@@ -56,3 +56,6 @@ jobs:
         az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/$(setVersionInfo.Version_Prefix)/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment
         az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment.pdb --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/$(setVersionInfo.Version_Prefix)/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment.pdb
         az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment.md5 --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/$(setVersionInfo.Version_Prefix)/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment.md5
+        az storage blob upload --account-name azureiiot --container-name binaries -f Microsoft.Azure.IIoT.Deployment --name ${{ replace(variables['Build.SourceBranch'], 'refs/heads/', '') }}/latest/${{ parameters.runtime }}/Microsoft.Azure.IIoT.Deployment
```

Comment:
Is the "latest" the last released version (2.7.x) or the "preview" latest(2.8.x)?
Are we following the same versioning schema/conventions like for the ACR (or MCR?) images of the various components of the platform?

Reply:
Good point. `latest` here will be most recent build, which will also be the one with highest version number. So this will be synonymous to `preview` tag that we use for images. I will rename it to preview. Thanks.